### PR TITLE
Add home.spec.js

### PR DESCRIPTION
**Pull request by Bryntseva**

### CODE REVIEW

File: test/specs/home.spec.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    it.only('T-3.002 Verify that favicon is present', () => {
+        expect(HomePage.favicon).toBeDisplayed();
+    });
+
+    it('T-3.003 Verify that favicon is present', () => {
+        expect(HomePage.favicon).toHaveAttr('alt','Local Coding Logo');
+    });
+
+
```
Comment (IrinaDash) [inline test/specs/home.spec.js:23]: extra empty line

File: test/specs/home.spec.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    it('T-3.001 Verify that title is "Local Coding"', () => {
+        expect(HomePage.localCodingLogo).toHaveText(expectedValue.texts.localCodingLink);
+    });
+
+    it.only('T-3.002 Verify that favicon is present', () => {
+        expect(HomePage.favicon).toBeDisplayed();
+    });
+
+    it('T-3.003 Verify that favicon is present', () => {
```
Comment (IrinaDash) [inline test/specs/home.spec.js:19]: is it copy-paste from T-3.002? need "The favicon is a Local Coding logo"

**Review by IrinaDash — CHANGES_REQUESTED** [review]

File: test/specs/home.spec.js (hunk 1 above)
Comment (Bryntseva) [reply test/specs/home.spec.js:26]: ok

File: test/specs/home.spec.js (hunk 2 above)
Comment (Bryntseva) [reply test/specs/home.spec.js]: ok

**Review by ayessetova — APPROVED** [review]

**Pull request merged**